core-bundle/src/ServiceAnnotation/Cron.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\ServiceAnnotation;
+
+use Doctrine\Common\Annotations\Annotation\Attribute;
+use Doctrine\Common\Annotations\Annotation\Attributes;
+use Doctrine\Common\Annotations\Annotation\Target;
+use Terminal42\ServiceAnnotationBundle\Annotation\ServiceTagInterface;
+
+/**
+ * Annotation to register a Contao cron.
+ *
+ * @Annotation
+ * @Target({"METHOD"})
+ * @Attributes({
+ *     @Attribute("value", type="string", required=true),
+ *     @Attribute("priority", type="int"),
+ *     @Attribute("cli", type="bool"),
```

Comment:
The `cli` attribute has to be renamed to `cli_only` in this class I think?

```suggestion
 *     @Attribute("cli_only", type="bool"),
```